src/components/common/MaintenanceWOForm.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react';
+import 'antd/dist/antd.css';
+import { getAuthHeader } from '../../api/index';
+import axios from 'axios';
+import { useOktaAuth } from '@okta/okta-react';
+import { postWO } from '../../api/index';
+
+import { Form, Input, Button, Select } from 'antd';
+
+function WorkOrderPage({ LoadingComponent }) {
+  //State
+  const [workOrder, setWorkOrder] = useState({
+    title: '',
+    description: '',
+    company: 1,
+    property: 1,
+    createdBy: '00ulthapbErVUwVJy4x6',
+    assignedTo: '00ulthapbErVUwVJy4x6',
+    priority: 2,
+    status: 1,
+  });
+
+  //Auth for api call
+  const { authState } = useOktaAuth();
+
+  //Handle changes
+
+  const handleChange = e => {
+    setWorkOrder({ ...workOrder, [e.target.name]: e.target.value });
+  };
+
+  function handleDropdownPriority(value, e) {
+    setWorkOrder({ ...workOrder, priority: value });
+  }
+
+  function handleDropdownStatus(value, e) {
+    setWorkOrder({ ...workOrder, status: value });
+  }
+
+  const handleSubmit = e => {
+    postWO(authState, workOrder);
+  };
+
+  return (
+    <>
+      <Form
+        labelCol={{
+          span: 4,
+        }}
+        wrapperCol={{
+          span: 14,
+        }}
+        layout="horizontal"
+        initialValues={{}}
+      >
+        <Form.Item
+          label="Work Order Title"
+          help="Maximum of 150 characters allowed."
+          className="WorkOrderTitle"
+        >
+          <Input
+            type="text"
+            name="title"
+            maxLength="150"
+            onChange={handleChange}
+          />
+        </Form.Item>
+        <Form.Item label="Date">
+          <Input
+            type="text"
+            name="date"
+            value={new Date().toISOString().slice(0, 10)}
+            placeholder="yyyy-mm-dd"
+            onChange={handleChange}
+          />
+        </Form.Item>
+        <Form.Item label="Priority">
+          <Select name="priority" onChange={handleDropdownPriority}>
+            <Select.Option value={0}>Critical</Select.Option>
+            <Select.Option value={1}>High</Select.Option>
+            <Select.Option value={2}>Medium</Select.Option>
+            <Select.Option value={3}>Low</Select.Option>
+          </Select>
+        </Form.Item>
+        <Form.Item label="Status">
+          <Select name="status" onChange={handleDropdownStatus}>
```

Comment:
Very much a nit pick of mine, but would making an array containing these statuses and then iterate through them to make the select option via something like `.map` work as well and be a bit cleaner? Doesn't need to be changed to this but just something that comes to mind lol

Reply:
For the maintenance form, I plan to redesign the form and make it more simple since maintenance staff will only need to edit two fields instead of the six that are in this form.